services/LocationTracking.ts

```diff
@@ -71,6 +77,7 @@ export async function startRouteTracking() {
     else {
       console.log("Background location tracking already started");
     }
+    */
   }
   else {
     console.log("Location permission was not granted but proceeding anyway");
```

Comment:
Skal vi ha en popup her som sier at vi trenger tillatelse til posisjon for å kunne spore gjeteren? På samme måte som med kameraet

Reply:
Kom på nå at første gang vi spør om lokasjonstillatelse er i DMS på linje 31. Da tenker jeg at det er lurt å ha en popup ihvertfall der

Reply:
#174